.gitignore

```diff
@@ -72,6 +72,7 @@ C:\\nppdf32Log\\debuglog.txt
 /config/web_hooks.yml
 /config/rolling_updates.yml
 /config/paperclip.yml
+/config/currencies.yml
```

Comment:
There is nothing in config/examples/currencies.yml?
Would be good to add what you removed there.

Reply:
@hallelujah Lacking intentionally 😝 the only ones that we require are USD and EUR and they are here... examples is just an example, no? why do we need an example for this?

And about this comment:
![image](https://user-images.githubusercontent.com/11318903/82994730-e88aa300-a002-11ea-863a-046ae5fbabf4.png)

I disagree that we will lose them, for this: https://github.com/3scale/deploy/pull/662

Reply:
But on-premises needs it anyway or is it an intentional breaking change? Like next release upgrade will make billing fail on a unknown currency?

Reply:
We spoke in private and I will add them only to `openshift/system/config`